grouper/fe/forms.py

```diff
@@ -39,6 +39,25 @@ def __call__(self, form, field):
                 "{} does not match format 'MM/DD/YYYY'".format(field.data))
 
 
+class DaysTimeDeltaField(IntegerField):
+
+    def process_data(self, value):
+        """Converting from TimeDelta notation to a simple integer"""
```

Comment:
Docstrings should be written in imperative style. `"""Convert from…"""`

(I personally prefer the `"""Converts from """` style, but it looks like we mostly use the former. Except in cases where I wrote the code, it looks like. :P )